features/src/main/scala/com/salesforce/op/features/types/FeatureTypeFactory.scala

```diff
@@ -110,6 +110,8 @@ case object FeatureTypeFactory {
         if (value == null) FeatureTypeDefaults.IDMap else new IDMap(value.asInstanceOf[Map[String, String]])
       case t if t =:= weakTypeOf[IntegralMap] => (value: Any) =>
         if (value == null) FeatureTypeDefaults.IntegralMap else new IntegralMap(value.asInstanceOf[Map[String, Long]])
+      case t if t =:= weakTypeOf[NameStats] => (value: Any) =>
```

Comment:
are these all the places you need to add the `NameStats` type? how about `FeatureBuilder` and `TestFeatureBuilder`?